content/en/docs/concepts/services-networking/service.md

```diff
@@ -1398,20 +1398,13 @@ in-memory locking). Kubernetes also uses controllers to check for invalid
 assignments (eg due to administrator intervention) and for cleaning up allocated
 IP addresses that are no longer used by any Services.
 
-#### IP address ranges for `type: ClusterIP` Services {#service-ip-static-sub-range}
```

Comment:
You will need to modify the `content/en/blog/_posts/2022-05-03-kubernetes-release-1.24.md` file which references this section.